libs/community/tests/unit_tests/vectorstores/test_indexing_docs.py

```diff
@@ -82,6 +82,7 @@ def check_compatibility(vector_store: VectorStore) -> bool:
         "SurrealDBStore",
         "TileDB",
         "TimescaleVector",
+        "TencentVectorDB",
```

Comment:
can we update docs/docs/modules/data_connection/indexing.ipynb to list TencentVectorDB

Reply:
sure, now it's in the Compatible Vectorstores list.